frontend/static/js/app.js

```diff
@@ -131,7 +133,10 @@ Vue.component("correction-view", {
       catch (err) {
         this.errors.push(err.message);
       }
-    }
+    },
+    toggleCommentBox: function () {
+      this.hasCheckedComment = !this.hasCheckedComment;

```

Comment:
the code shouldn't know what control is used on the UI. so instead of hasCheckedComment should be renamed to addComment or useComment.